WcaOnRails/app/mailers/competitions_mailer.rb

```diff
@@ -22,6 +22,14 @@ def notify_users_of_results_presence(user, competition)
     )
   end
 
+  def notify_users_of_id_claim_possibility(user, competition)
+    @competition = competition
+    mail(
+      to: user.email,
```

Comment:
Quick thought: people *love* replying to automated emails. Have you thought about a reasonable `reply_to` header for this email? Maybe the delegates of the competition?